src/API/actions/fetchMovies.js

```diff
@@ -0,0 +1,42 @@
+import {fetchMovieDetails, fetchMoviesPending, fetchMoviesSuccess} from "../../store/movies/actions";
+import {get, baseUrl} from "../api";
+
+
+
+export const fetchMovies = (filterParams) => {
+  const filter = filterParams?.filter ? `?filter=${filterParams?.filter}` : '';
```

Comment:
As I see the filterParams includes only from `filter` value. 
Maybe it's better to simply pass the 'filter' attribute?

The action will be called without 'object wrapping'.

But it's just a suggestion, it's ok how it is now.